backend/src/main/java/com/redhat/cloud/notifications/recipients/itservice/pojo/request/ITUserRequest.java

```diff
@@ -0,0 +1,42 @@
+package com.redhat.cloud.notifications.recipients.itservice.pojo.request;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class ITUserRequest {
+
+    public By by;
+    public Include include;
+
+    public ITUserRequest(String accountId, boolean adminsOnly) {
+        final By by = new By();
+        AllOf allOf = new AllOf();
+        allOf.status = "enabled";
+        allOf.ebsAccountNumber = accountId;
+
+        if (adminsOnly) {
+            PermissionCode permissionCode = new PermissionCode();
+            permissionCode.value = "admin:org:all";
+            permissionCode.operand = "eq";
```

Comment:
`permissionCode` seems constant. Can we avoid rebuilding it each time we need to send a request?